### ISSUE TITLE
Better Web Design

### ISSUE THREAD

**Issue by Blue-Robin-Taken** [Mr-Ben-Merch-Store/Website#80]
# General web design things we need
This issue is for general web design things. Any pull requests should be linked to this issue so that we can have a framework for our web design.

## Make a better color scheme
Currently the website could look a bit better as it's currently just green and stuff (which is quite goofy)

## Make some web animations for transitions
The website currently doesn't have any transitions and would be greatly improved by some transitions (other than the one that we have for \<h1> tags)

## Accessibility 
It's important that our website is accessible to everyone (on all devices as well)

**Comment by RedCat888** [comment]
I don't think the color scheme is an issue, it's just that the design itself is basic. We could work on animations/cool effects or add actually content then change things up. I do think that we should modify the color scheme but not yet.

**Comment by Blue-Robin-Taken** [comment]
I agree with that partially so I'm going to rename the issue to be more general about the website's design and how we can improve it

**Comment by xiberxia** [comment]
https://github.com/Mr-Ben-Merch-Store/Website/commit/9c772abc9d1748b11416976b7531bb07bf224698

current commit

**Comment by RedCat888** [comment]
I might go work on some animations later.

**Comment by Blue-Robin-Taken** [comment]
Linked to https://github.com/Mr-Ben-Merch-Store/Website/pull/88

**Comment by LizzyEll** [comment]
Linked to #99
